exotic.py

```diff
@@ -1301,10 +1354,11 @@ def parse_args():
         # t.start()
         # check to make sure the target can be found in the exoplanet archive right after they enter its name
 
-        if not os.path.exists("eaConf.json"):
+        # Checks to see if the file exists or is over one week old to scrape/rescrape parameters (units in seconds)
+        if not os.path.exists("eaConf.json") or time.time() - os.path.getmtime('eaConf.json') > 604800:
```

Comment:
Scraping will now be time-based. If planetary parameters have been on desktop for a week, rescrape.

Reply:
Great idea!